src/elements/map-block.html

```diff
@@ -188,7 +188,7 @@ <h2>{$ mapBlock.title $}</h2>
                     'elementType': 'geometry',
                     'stylers': [
                       {
-                        'color': '#e55746',
```

Comment:
The weird color is the Red of android makers (the river has the AM blue) :/

Reply:
Ah ok, chais pas ça rends bizarre :/

Reply:
Ok, retire complètement la couleur dans ce cas, je m'occuperai du marker, c'est une image à part.
Et si tu veux tu peux faire la PR sur le gtihub du paug (sinon je repousserai les changment d'ici à celui du paug)